docs/user/03-11-customize_secret.md

```diff
@@ -0,0 +1,45 @@
+# Customize the Default Credentials and Access
+
+You can customize the `sap-btp-manager` Secret and manage your own SAP BTP Operator's default configuration.
```

Comment:
Does this mean "manage the default configuration of your own SAP BTP Operator", or "manage your own default configuration of the SAP BTP Operator"?
I assume the 2nd option, anyway I'd suggest phrasing it in an unambiguous way,